src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -14,14 +14,12 @@ class OPL.KoComponents.ViewModels.OrificeCalculator
         params: true
         message: OPL.OrificeCalculator.Config.Dictionaries.Messages.integerError
 
-    @operatingPressureWarningMessage = ko.pureComputed (->
-      if (@operatingPressure() > 200 and @operatingPressure() < 401)
-        return OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningMayResult
-      else if (@operatingPressure() > 400)
-        return OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningWillResult
-      else
-        return ''
-      ), @
+    @operatingPressureWarningMessage = ko.pureComputed =>
+      switch true
```

Comment:
I'm not very fond of the `switch true` although I understand why it's used here. You could alternatively assign the value to a var and return that.
```
grade = switch
  when score < 60 then 'F'
  when score < 70 then 'D'
  when score < 80 then 'C'
  when score < 90 then 'B'
  else 'A'
  return grade
```